src/main/java/com/mitchej123/hodgepodge/mixins/late/extratic/MixinPartsHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.mitchej123.hodgepodge.mixins.late.extratic;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import cpw.mods.fml.common.FMLLog;
+import glassmaker.extratic.common.PartsHandler;
+
+@Mixin(PartsHandler.class)
+public class MixinPartsHandler {
+
+    /**
+     * @author DrParadox
+     * @reason Breaks TC integration with TE Materials
+     */
+    @Redirect(
+            method = "addMatrials",
+            at = @At(value = "INVOKE", target = "Lglassmaker/extratic/common/PartsHandler;_addM3PreciousMaterials()V"),
+            remap = false)
+    private static void _disableM3PreciousMaterials() {
```

Comment:
Use `hodgepodge$` as prefix of modified method name. I. e.:
```suggestion
    private static void hodgepodge$disableM3PreciousMaterials() {
```